_includes/_boinc_1B.htm

```diff
@@ -1,63 +1,21 @@
 <section class="boinc-b grc-bgcolor grclink">
     <!--Start of GRC Feature section-->
     <div class="container">
-
-    <!--First row of features-->
-        <!--Step 3 banner-->
         <div class="row">
-            <div class="col-12 text-center" >
-                <h4 class="text-bold">
-                    Pros and Cons of Account Managers
-                </h4>
-            </div>
```

Comment:
these tags need to be closed

Reply:
Hmm I see 4 div opens and 4 div closes what am I missing? Is there a particular tool you suggest for double-checking tag opens/closes?

Reply:
Sorry, three divs, maybe this diff shows it wrong here's the final file in the pull request apologies for my noobness around this https://github.com/makeasnek/Gridcoin-Site/blob/makeasnek-edits/_includes/_boinc_1B.htm

Reply:
Ah, you are correct.